src/components/Filters.jsx

```diff
@@ -1,15 +1,57 @@
 import { useProducts } from "../context/product-context";
+import { useState, useEffect } from "react";
 
 export function Filters() {
+  const [isFilterOpen, setisFilterOpen] = useState(false);
   const { state, dispatch } = useProducts();
+
+  const toggleFilter = () => {
+    document
+      .getElementsByClassName("filters")[0]
+      .classList.toggle("mob-show-filters");
+    setisFilterOpen((prev) => !prev);
+  };
+
+  useEffect(() => {
+    isFilterOpen
+      ? (document.body.style.overflow = "hidden")
+      : (document.body.style.overflow = "auto");
+  }, [isFilterOpen]);
+
```

Comment:
Avoid using vanilla JS.
Instead, you can do it by toggling className based on ```isFilterOpen```

Reply:
Yes I have discussed with other members and succesfully implemented with custom wrapper div